presto-cassandra/src/main/java/com/facebook/presto/cassandra/NativeCassandraSession.java

```diff
@@ -348,14 +350,14 @@ private static void checkColumnNames(List<ColumnMetadata> columns)
     {
         Map<String, ColumnMetadata> lowercaseNameToColumnMap = new HashMap<>();
```

Comment:
Same here, we need to update the map name, it is not just `lowercaseNameToColumnMap` anymore.